.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build application
+
+on: [pull_request, push]
```

Comment:
This way, it will run the action twice. This also used to happen with Validate Gradle Wrapper, but I fixed it.

Reply:
I think it's enough to run for pull requests. Or, if you want to run for push, run only for pushes to develop (although it's blocked for push, just like master). Otherwise, it will flood with logs, just like what was happening with the Validate Gradle Wrapper.